Automattic/wp-calypso
# [Draft] Woo Tailored Onboarding: Add/domain to ecommerce flow

## This is work in progress!

* This PR builds on top of https://github.com/Automattic/wp-calypso/pull/68722 
* Adds Domains steps to the new `ecommerce` flow
* Adds "waitForAtomic" step

#### Testing Instructions
* Make sure the flag `signup/tailored-ecommerce` is enabled in your environment
* Access `/setup/ecommerce` and you'll be redirected to `/setup/ecommerce/intro`
![Kzuj86.png](https://user-images.githubusercontent.com/3801502/199599954-b3782d86-09e7-476d-8438-1deeb554b45a.png)
* Click create your store and you'll be redirected to `storeProfiler`
![hi6n1t.png](https://user-images.githubusercontent.com/3801502/199600105-22c535f9-40d2-4722-9125-1ab581bbdcf9.png)
* Click Continue and you'll be redirected to `designCarousel` step
![zjAGwD.png](https://user-images.githubusercontent.com/3801502/199600270-b58849e5-e9de-4f1f-91f0-9c1800953a02.png)
* Choose a theme and click Continue.


* Pick a domain
![jBqZvh.png](https://user-images.githubusercontent.com/3801502/199599045-9f548ac2-fc6c-47de-8114-d08ac694504e.png)

* You'll go to checkout
![fRQXUo.png](https://user-images.githubusercontent.com/3801502/199599545-785e4cc9-e768-4655-9a16-20770a943b46.png)
* After checkout, the site is gonna be set up 
![image](https://user-images.githubusercontent.com/3801502/201522232-03d3fdc0-2403-40fa-8f2e-1b0c93f8a8e0.png)

* You should be redirected to `/home/<new-site-slug>`. 
**The Site should have the plan/domain/theme you chose in the flow.**
